changelog/13047.bugfix.rst

```diff
@@ -0,0 +1 @@
+* Fixed an issue in equality checks where `bool` and `numpy.bool_` types were not handled consistently.
```

Comment:
I think it might be good for this to be a bit more verbose:

- Mentioning that this is about `pytest.approx`
- perhaps mentioning that the behavior accidentally changed in 8.3.4 but is now restored to what 8.3.3 had
- plus maybe an example?

With this vague description, anyone reading the changelog who does something with numpy and bools will probably be concerned about whether this affects them or not.

Reply:
Good catch.

Thanks @BahramF73 for updating it.

Took the liberty of improving it further. 👍